site2/docs/reference-metrics.md

```diff
@@ -416,6 +416,50 @@ All the Pulsar Functions metrics are labelled with the following labels:
 | pulsar_function_last_invocation | Gauge | The timestamp of the last invocation of the function. |
 | pulsar_function_received_total | Counter | Total number of messages received from source. |
 | pulsar_function_received_total_1min | Counter | Total number of messages received from source in the last 1 minute. |
+pulsar_function_user_metric_ | Summary|User defined metric.
+
+## Connectors
+
+All the Pulsar connector metrics are labelled with the following labels:
+
+- *cluster*: `cluster=${pulsar_cluster}`. `${pulsar_cluster}` is the cluster name that you have configured in the `broker.conf` file.
+- *namespace*: `namespace=${pulsar_namespace}`. `${pulsar_namespace}` is the namespace name.
+
+Connector metrics contain **source** metrics and **sink** metrics.
+
+- **Source** metrics
+
+  | Name | Type | Description |
+  |---|---|---|
+  pulsar_source_written_total|Counter|Total number of records written to a Pulsar topic.
```

Comment:
I think it's better to use "the total number of...", same comments for the following similar descriptions.